test/scaleTest.js

```diff
@@ -0,0 +1,36 @@
+/* global suite test assert */
+
+function targetTester (keyframeA, keyframeB, timing) {
+  var targetA = document.createElement('div');
+  document.body.appendChild(targetA);
+
+  var targetB = document.createElement('div');
+  document.body.appendChild(targetB);
+
+  var animation = targetA.animate(keyframeA, timing);
+  animation.currentTime = 1000;
+  var result = getComputedStyle(targetA).transform;
+
+  var animation = targetB.animate(keyframeB, timing);
+  animation.currentTime = 1000;
+  var expected = getComputedStyle(targetB).transform;
+
+  return result === expected;
+}
+
+suite('transforms', function () {
+  var timing = {duration: 2000, iterations: 5, direction: 'alternate', easing: 'linear'};
+  test('scaleTransform', function () {
+    var isEqual = targetTester({scale: [0.5, 2.5]}, {transform: ['scale(0.5)', 'scale(2.5)']}, timing);
+    assert.equal(isEqual, true);
+
+    isEqual = targetTester({scale: ['', '']}, {transform: ['scale()', 'scale()']}, timing);
```

Comment:
scale() is not a valid value. The testing helper function should verify that the expectation transform values are valid.

You can do this by setting the value on an inline style, if the value does not get set then that tells you it was not valid.